ext_tables.php

```diff
@@ -132,7 +132,8 @@
         'solr::clearSolrConnectionCache',
         'ApacheSolrForTypo3\\Solr\\ConnectionManager->updateConnections'
     );
-
+}
+if ((TYPO3_MODE == 'BE') || ((TYPO3_MODE == 'FE') && isset($_POST['TSFE_EDIT']))) {
```

Comment:
Can you make two changes here: 
1) You don't need the braces around `TYPO3_MODE == 'FE'`
2) Please use triple equal signs for comparison `===`